app/move/controllers/view/view.update.links.js

```diff
@@ -0,0 +1,23 @@
+const getUpdateLinks = (updateSteps, t, urls) => {
+  const updateLinks = {}
+  updateSteps.forEach(updateJourney => {
+    const category = updateJourney.key
+    if (!urls[category]) {
+      return
+    }
+    const categoryText = t('moves::update_link.categories.' + category)
```

Comment:
I was under the impression that we enforced the use of html template strings for string concatenation, but I might be wrong.

Reply:
I think its the preferred pattern but maybe we're missing an eslint rule for it...

Reply:
We don't enforce it - but it is better